chapter07/requirements.txt

```diff
@@ -0,0 +1,5 @@
+apache-airflow
+minio
+pandas
+apache-airflow-providers-amazon
+sagemaker
```

Comment:
would it make sense to do some version pinning here? Just to avoid us not being able to run this in a not too distant future when one of these things breaks...